CMakeLists.txt

```diff
@@ -5,6 +5,7 @@ option(CI_BUILD "Set when building in CI. Enables -Werror where possible" OFF)
 option(ASAN "Compile with address sanitizers" OFF)
 option(QML_DEBUGGING "Enable qml debugging" OFF)
 option(COMPILE_QML "Compile Qml. It will make Nheko faster, but you will need to recompile it, when you update Qt." OFF)
+option(MAN "Build man page" ON)
```

Comment:
I would just set this similarly to how we do it for the VOIP flag:

```
set(VOIP_DEFAULT ON)
if (APPLE OR WIN32)
	set(VOIP_DEFAULT OFF)
endif()
```

Maybe `UNIX AND NOT APPLE`

Users can still override it, if they want in that case. But .dmg's on macOS usually ship no man page, neither do flatpaks, afaik. So it makes sense to disable it by default on macOS at least.

Reply:
Actually, a few parts of the manpage look to be UNIX AND NOT APPLE specific to me, like the file locations.